src/assets/db/gsoc2016.json

```diff
@@ -0,0 +1,134 @@
+[
+ {
+   "PROJECT": "Malaria (Android)",
+   "STUDENT": "Yatna (India)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Ankita (India); Leroi"
+ },
+ {
+   "PROJECT": "Malaria (iOS)",
+   "STUDENT": "Teodor (Romania)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Bruno (Portugal); Jayaruwan"
+ },
+ {
+   "PROJECT": "Malaria (Web)",
+   "STUDENT": "Ananta (India)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Khushali; Ritesh; Rheeya"

```

Comment:
Khushali - USA, Ritesh - USA, Rheeya-USA
All the 3 are currently based out of USA as per their Linkedin, but they are from India.